### PR TITLE
Add functionality to mobile menu

### PR DESCRIPTION
**Pull request by Mukumbuta**
### features implemented:
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,30 +23,30 @@
...
               <img
                 class="nav-email"
-                src="./icons/email-icon.svg"
+                src="./images/email-icon.svg"
                 alt="email icon"
               />
             </a>
           </li>
         </ul>
       </nav>
```
Comment (codecaiine) [inline index.html:44]: - [ ] Well done, when I click on a menu option, it leads me to the part mentioned (example: **About/Portfolio/contact**) :100: But it's impossible to scroll up or down afterward. I have to refresh the page, please fix this to give the user a pleasant user experience. :smile: 

https://www.loom.com/share/cc06d0baaae34627b6a9cd7d219be9b1

File: index.html [hunk 2]
```diff
@@ -26,30 +23,30 @@
...
+            <a href="#contact">
               <img
                 class="nav-email"
-                src="./icons/email-icon.svg"
+                src="./images/email-icon.svg"
                 alt="email icon"
               />
             </a>
           </li>
         </ul>
```
Comment (codecaiine) [inline index.html:43]: -  [ ] Good job you implemented the menu :+1:  However there is some issue you need to address with your menu design. :cry:  You are using the wrong  **font-size**  and **font-weight** and the position for your menu option. Please fix it in order to have the similar to the Figma template :smile: 

- [ ] Please adjust your menu option **underlines** to make it **thinner** and use the same **color** from the Figma design, cause it is very important to follow the design from the chosen template in Figma, peer attention to the details is also important  :smile: 

You can check the different screenshots below to check the difference :point_down: 

### Your project
Screenshot from 2022-07-19 12-09-49

### Figma template
Screenshot from 2022-07-19 12-09-32

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
- Clean code :heavy_check_mark:  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**